src/tests/example.spec.js

```diff
@@ -24,4 +26,56 @@ test.describe('', () => {
         await shopingCartPage.removeCartItemById(0);
         await expect(shopingCartPage.cartItems).not.toBeAttached();
     });
+
+    test('check backward sorting', async ({ loginPage, inventoryPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+        const itemsBefore = await inventoryPage.inventoryItemNames.allInnerTexts();
+        const itemsSorted = itemsBefore.sort().reverse();
+        await inventoryPage.selectSorting('Name (Z to A)');
+        const itemsAfter = await inventoryPage.inventoryItemNames.allInnerTexts();
+        expect(itemsAfter).toEqual(itemsSorted);
+    });
+
+    test('Add product to the cart and compare', async ({ loginPage, inventoryPage, shopingCartPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+        const selectedCount = 3;
+        const sortedProducts = await inventoryPage.selectNItems(selectedCount);
+        expect(await inventoryPage.getNumberOfItemsInCart()).toBe(selectedCount.toString());
+
+        await inventoryPage.shopingCart.click();
+        for (let i = 0; i < selectedCount; i++) {
```

Comment:
Just for learning purposes - let's try to implement this test in another manner -> collect the data in the shopping cart into an array and compare it with the initial array